dhis2-mobile-program-rules/src/main/java/org/dhis2/mobileProgramRules/RulesRepository.kt

```diff
@@ -131,7 +131,7 @@ class RulesRepository(private val d2: D2) {
                             organisationUnitCode = d2.organisationUnitModule().organisationUnits()
                                 .uid(
                                     event.organisationUnit(),
-                                ).blockingGet()!!.code(),
+                                ).blockingGet()?.code() ?: "",
```

Comment:
`organisationUnitCode` is type `String?` so, why `?: ""` and not letting to be null if it is the case just changing `""` by `?`